pom.xml

```diff
@@ -68,6 +68,7 @@
         <version.org.apache.directory.server>2.0.0-M15</version.org.apache.directory.server>
         <version.org.apache.httpmime>4.2.6</version.org.apache.httpmime>
         <version.org.apache.httpcomponents>4.2.6</version.org.apache.httpcomponents>
+        <version.org.apache.maven.plugins.maven-surefire-plugin>2.21.0</version.org.apache.maven.plugins.maven-surefire-plugin>
```

Comment:
No!

Either upgrade jboss parent to latest or override same property name that is already in use

Reply:
Okay, sorry. I did not noticed this is inherited from jboss-parent. I have changed the property name so it follows format of other `version` properties. I will ammend this.

@stuartwdouglas is it ok from your point of view if I update the `jboss-parent` to latest 26 version?